src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propsList = sourceString.split(';');
+  const propsObject = {};
+
+  propsList.forEach(property => {
+    const currentProp = property.split(':');
+
+    if (currentProp[0] && currentProp[1]) {
+      propsObject[currentProp[0].trim()] = currentProp[1].trim();
+    }
+  });
```

Comment:
forEach is like `for` loop 🙂 Let's learn how to use reduce

Reply:
done! is it better to use reduce here just to learn?) or there is some other reason?

Reply:
Reduce is more logical here if we using tools that supposed for specific operation

Of course you can do everything using loop (or forEach), and it will works fine, but why, if we have tools for it 🙂 

For example here we have array and we want to make object from it (to get one instance from list of instance) - it's exactly why reduce is needed. (also using reduce we don't need any empty variable outside our method)

Use cases of forEach - is when you need to do some operation with element and you don't need to get result from it (just like replacement of for loop, it will be more often used in DOM, React, promises, fetch, etc. 🙂 ) 

And of course this case is good for learning of reduce usage (and understanding that reduce can be used in many cases, not just to calculate sum of numbers)